src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys-sphinx-theme-variable.css

```diff
@@ -54,6 +54,14 @@
 * table font sizes
 */
   --pst-table-font-size: 16px;
```

Comment:
Table header should be 12 px and table cell texts should be 14px

Reply:
ok i will add.

Reply:
updated